lib/file-vault-utils.js

```diff
@@ -31,7 +31,7 @@ const auth = async () =>
       .catch(error => {
         // eslint-disable-next-line no-console
         logger.log('error', 'Keycloak authentication error');
-        reject(error);
+        reject(new Error('Keycloak authentication error', { cause: error }));
```

Comment:
just minor point, you started with Error in each log above, is it worth doing the same here to be consistent?

Reply:
The error above is for the Winston logger to colourise red whereas the new error is used for the Notify email and has pretty good context around it already in the template.